README.md

```diff
@@ -56,29 +56,14 @@ For more information on how to contribute, please see our [Contribution Guide](d
 Use these directions if you want to build the source on your machine.
 If you just want to run the system, see "Run the Code" below.
 
-## macOS
-Ensure your development environment is set correctly for clang:
-
-`sudo xcode-select -switch /Library/Developer/CommandLineTools`
-
-Or, if you've changed this in the past, you can reset to point to commandline tools with:
-
-`sudo xcode-select --reset`
-
-
-1. Install dependencies: `brew install leveldb llvm@11 googletest lcov make wget cmake`
-2. `./scripts/configure.sh`
-3. `./scripts/build.sh`
-
-Note: To run clang-tidy and clang-format (required by `lint.sh`), you must add them both to your path.
-
-Ex: `ln -s /usr/local/opt/llvm@11/bin/clang-tidy /usr/local/bin/clang-tidy`
-
-## Linux
-
 1. `./scripts/configure.sh`
 2. `./scripts/build.sh`
 
+## macOS
+Note that if you have not already installed the xcode cli tools you will need to:
+
+`xcode-select --install`
```

Comment:
The original PR probably should have had code snippets like this as full code blocks (as we do most elsewhere in the readme). Can that be done here?

Reply:
Sounds good.
I also added blocks to the build steps themselves so they should all match.
Let me know if you actually wanted the configure and build commands to stay the way they were.